model/src/test/kotlin/ProjectTest.kt

```diff
@@ -20,24 +20,48 @@
 package com.here.ort.model
 
 import io.kotlintest.shouldBe
-import io.kotlintest.specs.StringSpec
+import io.kotlintest.specs.WordSpec
 
 import java.io.File
 
-class ProjectTest : StringSpec({
-    "collectDependencies contains all dependencies" {
-        val expectedDependencies = listOf(
-                "Maven:junit:junit:4.12",
-                "Maven:org.apache.commons:commons-lang3:3.5",
-                "Maven:org.apache.commons:commons-text:1.1",
-                "Maven:org.apache.struts:struts2-assembly:2.5.14.1",
-                "Maven:org.hamcrest:hamcrest-core:1.3"
-        )
-
-        val analyzerResultsFile =
-                File("../analyzer/src/funTest/assets/projects/synthetic/gradle-expected-output-lib.yml")
-        val project = analyzerResultsFile.readValue<ProjectAnalyzerResult>().project
-
-        project.collectDependencies().map { it.id.toString() } shouldBe expectedDependencies
+class ProjectTest : WordSpec({
+    "collectDependencies" should {
+        "get all dependencies by default" {
+            val expectedDependencies = listOf(
+                    "Maven:junit:junit:4.12",
+                    "Maven:org.apache.commons:commons-lang3:3.5",
+                    "Maven:org.apache.commons:commons-text:1.1",
+                    "Maven:org.apache.struts:struts2-assembly:2.5.14.1",
+                    "Maven:org.hamcrest:hamcrest-core:1.3"
+            )
+
+            val analyzerResultsFile =
+                    File("../analyzer/src/funTest/assets/projects/synthetic/gradle-expected-output-lib.yml")
+            val project = analyzerResultsFile.readValue<ProjectAnalyzerResult>().project
+
+            project.collectDependencies().map { it.id.toString() } shouldBe expectedDependencies
+        }
+
+        "get no dependencies for a depth of 0" {
+            val analyzerResultsFile =
+                    File("../analyzer/src/funTest/assets/projects/synthetic/gradle-expected-output-lib.yml")
+            val project = analyzerResultsFile.readValue<ProjectAnalyzerResult>().project
+
+            project.collectDependencies(maxDepth = 0).map { it.id.toString() } shouldBe emptyList()
```

Comment:
`.map { it.id.toString() }` is superfluous.
You could also use `should beEmpty()`.

Reply:
Done.